docs/articles/documentation/getting-started/index.md

```diff
@@ -0,0 +1,195 @@
+---
+layout: docs
+title: Getting Started
+permalink: /documentation/getting-started/
+---
+# Getting Started
+
+This guide will provide you with step-by-step instructions on how to create a functional web test with TestCafe.
+You will learn how to install TestCafe, write a test, run the test on multiple browsers and view test results.
+During creating a test, you will learn the basics of TestCafe API, advancing step by step, from the simple to more elaborate testing techniques.
+
+All tests in this guide target a sample page available at [http://testcafe.devexpress.com/Example](http://testcafe.devexpress.com/Example). You can try running each of them against this page by yourself.
+
+This topic contains the following sections.
+
+* [Setting up Test Project](#setting-up-test-project)
+* [Installing TestCafe](#installing-testcafe)
+* [Writing Test](#writing-test)
+  * [General Code Structure](#general-code-structure)
+  * [Performing Actions on Page](#performing-actions-on-page)
+  * [Assertions](#assertions)
+  * [Dialog Handling](#dialog-handling)
+  * [Authentication](#authentication)
+* [Running Test](#running-test)
+* [Viewing Test Results](#viewing-test-results)
```

Comment:
We have such references in other topics: [Custom Reporter Plugin](https://github.com/DevExpress/testcafe/blob/master/docs/articles/documentation/extending-testcafe/custom-reporter/index.md), [Helper Methods](https://github.com/DevExpress/testcafe/blob/master/docs/articles/documentation/extending-testcafe/custom-reporter/helper-methods.md). 
We've discussed this with @VasilyStrelyaev and decided to add TOC to big topics.

> it should be in a menu
The topic's sections can be in the menu, if we divide it to separate topics/page. However, I think, GS should be a single page, so that users can go through the GS steps without additional clicks to switch to the next section (as we do for Custom Reporter Plugin)